lesson/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('lesson/', views.get_all_lessons, name='get_all_lessons'),
+    path('lesson/<str:subset>/', views.get_lessons_by_subset, name='get_lessons_by_subset'),
+    path('lesson/<str:subset>/<int:lesson_id>/', views.get_lesson_by_id, name='get_lesson_by_id'),
```

Comment:
I know this is a really small change request, but could you make it "lessons" instead of "lesson".  I know this has nothing to do with functionality, but it's just a part of API convention.